samples/sampler/stories/default/PopupMenu.js

```diff
@@ -0,0 +1,43 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/knobs';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import LabeledIconButton from '@enact/agate/LabeledIconButton';
+import PopupMenu from '@enact/agate/PopupMenu';
+
+const Config = mergeComponentMetadata('PopupMenu', PopupMenu);
```

Comment:
We need the `PopupMenuBase` instead of the `PopupMenu` here because all prop types and default props are defined not in the `PopupMenuBase` but in the `PopupMenu`.

```
import {PopupMenu, PopupMenuBase} from '@enact/agate/PopupMenu';

const Config = mergeComponentMetadata('PopupMenu', PopupMenuBase);
```

Reply:
Done, Thanks!
I also fixed Popup sampler